critiquebrainz/db/revision.py

```diff
@@ -0,0 +1,79 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def get(review_id):
+    """Get revisions on a review ordered by the timestamp
+
+    Args:
+        review_id (uuid): ID of review
+
+    Returns:
+        List of dictionaries of revisions of the review
+        with the following structure:
+        {
+            "id": (int),
+            "review_id": (uuid),
+            "timestamp": (datetime),
+            "text": (string)
+        }
+
+        and the number(count) of reviews
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT id, review_id, timestamp, text
+              FROM revision
+             WHERE review_id = :review_id
+          ORDER BY timestamp DESC
+        """), {
+            "review_id": review_id,
+        })
+
+        rows = result.fetchall()
+        if not rows:
+            raise db_exceptions.NoDataFoundException("Cannot find specified review.")
+        rows = [dict(row) for row in rows]
+        count = len(rows)
```

Comment:
Oh, sorry. I misunderstood this. There are places where total count should be calculated separately, but it's not the case here. This wasn't about the naming, `count` is a good name for that.